bitnami/external-dns/templates/psp-clusterrolebinding.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
 kind: ClusterRoleBinding
 metadata:
-  name: {{ template "external-dns.fullname" . }}-psp
```

Comment:
@joancafom  @FraPazGal 
I'm not sure if I should leave the `-psp` or remove it as well?

Reply:
In these cases, you need to add `-psp`. You can do so using like:

```
{{ printf "%s-%s" (include "common.names.fullname.namespace" .) "psp" }}
```

Reply:
I´ve added this line of code to wherever it seems it will fit.